go.mod

```diff
@@ -55,6 +55,7 @@ require (
 	github.com/acarl005/stripansi v0.0.0-20180116102854-5a71ef0e047d // indirect
 	github.com/asaskevich/govalidator v0.0.0-20230301143203-a9d515a09cc2 // indirect
 	github.com/chzyer/readline v1.5.1 // indirect
+	github.com/cilium/ebpf v0.9.1 // indirect
```

Comment:
This adds ... a lot of ... code.

Need to investigate

Reply:
https://github.com/opencontainers/runc/pull/3452 changed it in runc so we do not have to import so much, I guess there is a regression which added the unwanted imports back in some place.

Reply:
nevermind this is a downgrade, go versioning is something...
there is a reason this uses a replace here.

Reply:
Thanks for checking.  Let's close it